conversion.py

```diff
@@ -127,11 +128,10 @@ def adjustMarkdown(filename):
 
       # if all caps, then heading 1
       if allUpper(workingLines[i-1]):
-        if workingLines[i-1] != "NAME\n":
+        if "SEE ALSO" in workingLines[i-1]:
+            seeAlso = True
         #add a new line after each title
-          line = '\n# ' + workingLines[i-1].title()+'\n'
-        else:
-          line = '# ' + workingLines[i-1].title()+'\n'
+        line = '\n# ' + workingLines[i-1].title()+'\n'
```

Comment:
Are you sure this doesn't add extra lines? I am assuming you tested it, I just don't understand this fix lol